dask_ml/metrics/pairwise.py

```diff
@@ -113,13 +132,15 @@ def check_pairwise_arrays(X, Y, precomputed=False):
 
 
 @derived_from(metrics.pairwise)
-def linear_kernel(X, Y=None):
+def linear_kernel(X: ArrayLike, Y: Optional[ArrayLike] = None) -> ArrayLike:
     X, Y = check_pairwise_arrays(X, Y)
     return da.dot(X, Y.T)
 
 
 @derived_from(metrics.pairwise)
-def rbf_kernel(X, Y=None, gamma=None):
+def rbf_kernel(
+    X: ArrayLike, Y: Optional[ArrayLike] = None, gamma: Optional[int] = None
```

Comment:
```suggestion
    X: ArrayLike, Y: Optional[ArrayLike] = None, gamma: Optional[float] = None
```

I think it's a float.